### ISSUE TITLE
关于DBNet的一些问题

### ISSUE THREAD

**Issue by float123** [wang-xinyu/tensorrtx#397]
作者你好，感谢分享了trt的API代码，我在参考你的trt版dbnet的时候有一些疑惑，在构建FPN层的时候，为什么使用的是leakyrelu？pytorch的原代码用的是relu。并且batchnorm设置是1e-3，而不是pytorch的默认值1e-5？

这是pytorch中的ConvBnRelu步骤：
`self.bn = nn.BatchNorm2d(out_channels)`
`self.relu = nn.ReLU(inplace=inplace)`
这是trt版本中的convBnLeaky步骤：
`IScaleLayer* bn1 = addBatchNorm2d(network, weightMap, *conv1->getOutput(0), lname.substr(0, lname.find_last_of(".")) + bnname, 1e-3);`
`auto lr = network->addActivation(*bn1->getOutput(0), ActivationType::kLEAKY_RELU);`
`lr->setAlpha(0.1);`

**Comment by BaofengZan** [comment]
@float123 你好，我又看了代码，确实是写错了。我会更新下。谢谢

**Comment by float123** [comment]
> @float123 你好，我又看了代码，确实是写错了。我会更新下。谢谢

感谢你的回复，预祝春节快乐！
